docs/guides/http/json-validation.md

```diff
@@ -0,0 +1,93 @@
+# JSON validation
+
+Bocadillo has built-in support for JSON data validation using [TypeSystem]. It allows you to perform schema-based validation, as well as use [orm] to get RESTful validation _à la_ Django REST Framework.
+
+[typesystem]: https://www.encode.io/typesystem/
+[orm]: https://github.com/encode/orm
+
+## How it works
```

Comment:
@cs01 I'm not sure this guide is clear enough about how validation works (e.g. we catch `ValidationError` so people can just use TypeSystem without the hassle). Could you review this write up if you have some time?

Reply:
Sure! Thanks for writing it up! Can't promise when I'll be able to get to it though.